source/ServerCursor.cpp

```diff
@@ -86,8 +86,7 @@ void ServerCursor::process_cursor_resize([[maybe_unused]]uint32_t time)
     }
   else
     {
-      Output &output(server.outputManager.getOutput(view->getWlrOutput()));
-      wm::WindowTree &windowTree(output.getWindowTree());
+      wm::WindowTree &windowTree(server.getActiveWindowTree());
```

Comment:
Are your sure before and after are equivalent? A little comment could help here.